tiflash/tiflash-configuration.md

```diff
@@ -175,22 +174,27 @@ delta_index_cache_size = 0
     engine-addr = The external access address of the TiFlash coprocessor service.
 [raftstore]
     ## Specifies the number of threads that handle snapshots.
-    ## The default number is 2. 
+    ## The default number is 2.
     ## If you set it to 0, the multi-thread optimization is disabled.
-    snap-handle-pool-size = 2 
+    snap-handle-pool-size = 2
 
     ## Specifies the shortest interval at which Raft store persists WAL.
     ## You can properly increase the latency to reduce IOPS usage.
     ## The default value is "4ms".
     ## If you set it to 0ms, the optimization is disabled.
     store-batch-retry-recv-timeout = "4ms"
+[security]
+    ## New in v4.0.10. This configuration item enables or disables log redaction.
```

Comment:
mark

Reply:
changed in https://github.com/pingcap/docs/pull/5000.